content/product-engineering/engineering/deployments/instances.md

```diff
@@ -13,11 +13,11 @@ For deployments of Sourcegraph we manage for customers, see [managed instances](
 
 Also see [playbooks](./playbooks.md) for common actions related to operating our Sourcegraph deployments.
 
-## sourcegraph.com
+## Sourcegraph Cloud
 
 [![Build status](https://badge.buildkite.com/ef1289610fdd05b606bf1e57a034af2365c7b09c95ac6121f9.svg)](https://buildkite.com/sourcegraph/deploy-sourcegraph-dot-com)
 
-This deployment is also colloquially referred to as "Sourcegraph Cloud", "Cloud", and "dot-com". It is the public deployment available to the public at [sourcegraph.com/search](https://sourcegraph.com/search).
+This deployment is also colloquially referred to as "sourcegrpah.com", "Cloud", and "dot-com". It is the public deployment available to the public at [sourcegraph.com/search](https://sourcegraph.com/search).
```

Comment:
Sourcegraph.com 😅

Reply:
OH NO

Reply:
I mean **_I_** refer to it exclusively as `sourcegrpah.com`, but here's a fix anyway: https://github.com/sourcegraph/handbook/pull/1308